runtime/src/accounts_db.rs

```diff
@@ -2265,15 +2296,108 @@ impl AccountsDb {
         self.accounts_index.all_roots()
     }
 
+    fn select_candidates_by_total_usage(
```

Comment:
Nit: Maybe a comment on what this function does/returns?

Reply:
Will do